### PR TITLE
PP-5389 Ledger transactions search (basic)

### PR DESCRIPTION
**Pull request by sfount**
* basic transactions search - redirect to transactions view (lookup by reference etc. to be added)
* search template 
* platform admin override query
* routes added for search

### CODE REVIEW

File: src/assets/payuk-toolbox.scss [hunk 1]
```diff
@@ -116,12 +116,24 @@
...
+  background-color: govuk-colour("turquoise");
+}
+
+.status-failed {
+  background-color: govuk-colour("orange");
+}
+
+.status-in-progress {
+  background-color:	#6f777b;
+  color: 	#f3f2f1;
```
Comment (jankowiakmaria) [inline src/assets/payuk-toolbox.scss]: that's quite odd formatting

File: src/tests/fixtures/payment.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import {
   Transaction,
-  TransactionStatus,
   PaymentProvider,
-  CardBrand
+  CardBrand,
+  TransactionStatus
```
Comment (jankowiakmaria) [inline src/tests/fixtures/payment.ts]: If you're changing the order, I prefer alphabetical.

File: src/web/modules/transactions/transactions.http.ts [hunk 3]
```diff
@@ -5,11 +5,28 @@ import { Transaction } from 'ledger'
...
+  }
+}
+
 export async function show(req: Request, res: Response, next: NextFunction): Promise<void> {
   try {
     const transaction = await Ledger.transaction(req.params.id) as Transaction
     const account = await Connector.account(transaction.gateway_account_id)
     const service = await AdminUsers.gatewayAccountServices(transaction.gateway_account_id)
+
+    console.log(JSON.stringify(transaction))
```
Comment (jankowiakmaria) [inline src/web/modules/transactions/transactions.http.ts]: do you want to leave this crude `console.log`?

Comment (sfount) [reply src/web/modules/transactions/transactions.http.ts]: added very quickly, removed following

**Pull request merged**